server/service/handler.go

```diff
@@ -445,6 +445,10 @@ func attachFleetAPIRoutes(r *mux.Router, svc fleet.Service, config config.FleetC
 	ne.POST("/api/v1/fleet/sso", initiateSSOEndpoint, initiateSSORequest{})
 	ne.POST("/api/v1/fleet/sso/callback", makeCallbackSSOEndpoint(config.Server.URLPrefix), callbackSSORequest{})
 	ne.GET("/api/v1/fleet/sso", settingsSSOEndpoint, nil)
+
+	// Fleet Sandbox demo login (always errors unless FLEET_DEMO environment variable is set)
+	ne.POST("/api/_version_/fleet/demologin", makeDemologinEndpoint(config.Server.URLPrefix), demologinRequest{})
```

Comment:
This is a sort of "login endpoint", should it be rate-limited like `/api/_version_/fleet/login`?
https://github.com/fleetdm/fleet/blob/ade216a723625f34c2ea07191f837fae48e164b4/server/service/handler.go#L460-L466